neighsyncd/neighsyncd.cpp

```diff
@@ -26,17 +30,32 @@ int main(int argc, char **argv)
         {
             NetLink netlink;
             Select s;
+            time_t starttime;
 
-            netlink.registerGroup(RTNLGRP_NEIGH);
-            cout << "Listens to neigh messages..." << endl;
-            netlink.dumpRequest(RTM_GETNEIGH);
-
-            s.addSelectable(&netlink);
             if (sync.getRestartAssist()->isWarmStartInProgress())
             {
                 sync.getRestartAssist()->readTableToMap();
+                starttime = time(NULL);
```

Comment:
I think you want to use clock_gettime in case ntp skews the clock during warm reboot.

Reply:
Or better use std::chrono::steady_clock